### PR TITLE
Features/background check

### PR DESCRIPTION
**Pull request by RomainDesmidt**
created a component to call from each pages so that it would periodically check for geolocation coordinates, and would create records in firebase garden if current loc is in an undiscovered garden

### CODE REVIEW

File: src/app/components/background-check/background-check.component.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      next: (coords: any) => {
+          this.myLong = coords.lon;
+          this.myLat = coords.lat;
+      },
+      error: (error: string) => {
+        console.error(error);
+      },
+    });
+  }
+  */
```
Comment (KevinRoig) [inline src/app/components/background-check/background-check.component.ts:103]: Dead code, Bastien is going to kill you!

**Review by KevinRoig — CHANGES_REQUESTED** [review]
See comments please

File: src/app/components/background-check/background-check.component.ts (hunk 1 above)
Comment (RomainDesmidt) [reply src/app/components/background-check/background-check.component.ts:103]: Actually that's on purpose, not something forgotten ! might i remind you we are doing this with test function, i kept the function we would use if we remake it without spoof coords but using current position instead !

**Comment by RomainDesmidt** [comment]
See my own comment please, i can also play the link to game !

File: src/app/components/background-check/background-check.component.ts (hunk 1 above)
Comment (KevinRoig) [reply src/app/components/background-check/background-check.component.ts:103]: Oh, OK, I did not see the differences.

**Review by KevinRoig — APPROVED** [review]
Ok

**Review by bastienapp — APPROVED** [review]
Validé

**Pull request merged**